holycorn.c

```diff
@@ -298,8 +298,10 @@ static TupleTableSlot * rbIterateForeignScan(ForeignScanState *node) {
             slot->tts_isnull[i] = true;
             break;
           case MRB_TT_FALSE: //TODO: use BoolGetDatum
-            slot->tts_isnull[i] = true;
-            slot->tts_values[i] = BoolGetDatum(false);
+            if (!mrb_nil_p(column)) {
+                slot->tts_isnull[i] = true;
+                slot->tts_values[i] = BoolGetDatum(false);
+            }
```

Comment:
awesome, thanks!

Should we add an `else` branch if it's `mrb_nil_p` returns `true` ?

Not sure how to answer your question about `slot->tts_isnull[i]`  but given what we've seen, it should probably be true if it's `mrb_nil_p` returns `true`, else `false.

What do you think?